### ISSUE TITLE
[Bug][a11y] The page after login(Connected) cannot be dismissed using TalkBack

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#3578]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Launch Fenix.
1. Login with valid credentials.

### Expected behavior
The `Connected` page can be closed accessing the `X` element from the page after a login.

### Actual behavior
When the login is successfully made the `Connected` page cannot be dismissed as the `X` and the top elements from the page cannot be accessed with Talkback. ### Device information

* Android device: **`Samsung Galaxy Note 9(Android 8.1.0).`**
* Fenix version: **`6/19 1.0.0-rc.2`** 
    1. Notes
* [video](https://drive.google.com/file/d/1s4ZQl-pRR4-gR0jU-5Zr4WtHahhreD0L/view?usp=sharing)
At the end I selected manually with double tap the top bar but with a swipe cannot be accessed.

**Comment by abodea** [comment]
Please note that this issue is also reproducible if accessing by mistake `Turn on Sync` or `Create an account` there is no option to go back from those menus as the `X` cannot be selected.

**Comment by ekager** [comment]
We auto close these now once you've been authenticated. Closing this. Please reopen if I misunderstood.
